### DISCUSSION TITLE
Make pinned tabs only show up in their current workspace and not globally

### DISCUSSION THREAD

**Discussion by lukamrkonjic** [Floorp-Projects/Floorp#1444] [Feature Request]
I'm loving the workspace behavior in Floorp, but dislike that pinned tabs are global. Wish we had the ability to pin tabs within a workspace and not make them globally visible in all workspaces.

Thank you!

**Comment by ledsales** [comment]
This, absolutely this, having pinned tabs appear in every workspace is completely pointless and no other browser I've used (Opera, Vivaldi) does it this way, pinned tabs are treated like regular tabs in respect to workspaces, the only appear in the workspace they were pinned to.

This issue alone will stop me using floorp as my primary browser, I have dozens of pinned tabs spread across several workspace and they need to be workspace-specific, not global.

**Comment by Herbert-Sg** [comment]
That would be really great! It shouldn't be that much effort 🙏

**Comment by ioc2e3** [comment]
> ... and they need to be workspace-specific, not global.

Agree ...

They need to be workspace-specific ...

.
